src/LaratrustServiceProvider.php

```diff
@@ -194,6 +194,10 @@ protected function offerPublishing()
             $this->publishes([
                 __DIR__. '/../config/laratrust_seeder.php' => config_path('laratrust_seeder.php'),
             ], 'laratrust-seeder');
+            
+            $this->publishes([
+                __DIR__. '/../resources/views/panel' => resource_path('views/vendor/laratrust'),
```

Comment:
here the destination path has to include `panel` so it would be:
```php
__DIR__. '/../resources/views/panel' => resource_path('views/vendor/laratrust/panel'),
```

Reply:
Done and done. Thanks.